### PR TITLE
chore: edited njk templates and added readme info for datalayer and header

### PR DESCRIPTION
**Pull request by davidornelas11**
Edited njk templates to coorrectly import the header and dataLayer scripts.
Added more info to readme to show how to add scrupts to standalone application

### Checklist

- [x] Unity project successfully builds from root `yarn install` & `yarn build`
- [x] Add/updated examples
- [x] No new console errors
- [x] Accessibility checks

### CODE REVIEW

File: server/views/asuheader/index.njk [hunk 1]
```diff
@@ -7,9 +7,9 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <!-- Bootstrap includes for asu-web-standards-bootstrap4-->
   <link rel="stylesheet" href="../@asu/unity-bootstrap-theme/css/unity-bootstrap-theme.bundle.css" type="text/css" media="all">
-  <script type="text/javascript" src="./@asu/unity-bootstrap-theme/js/bootstrap.bundle.min.js"></script>
-  <script type="text/javascript" src="./@asu/unity-bootstrap-theme/js/data-layer.js"></script>
-  <script type="text/javascript" src="./@asu/unity-bootstrap-theme/js/global-header.js"></script>
+  <script type="text/javascript" src="../@asu/unity-bootstrap-theme/js/bootstrap.bundle.min.js"></script>
```
Comment (scott-williams-az) [inline server/views/asuheader/index.njk:10]: Nice catch! I missed this.

File: server/storybook-index-generator.js [hunk 2]
```diff
@@ -281,11 +281,12 @@ module.exports = generateHTML = packages => `
       </div>
     </div>
   </footer>
-
+  <--- Initialize DataLayer script and add scrolled behavior for header --->
```
Comment (scott-williams-az) [inline server/storybook-index-generator.js]: These comments are left over from me.
Can you clean up my mess, Please :)

File: server/storybook-index-generator.js [hunk 3]
```diff
@@ -281,11 +281,12 @@ module.exports = generateHTML = packages => `
...
   <script>
     window.initDataLayer();
     window.initGlobalHeader();
   </script>
+  <--- End header script script --->
```
Comment (scott-williams-az) [inline server/storybook-index-generator.js]: This comment also can be removed

**Review by scott-williams-az — APPROVED** [review]

**Review by scott-williams-az — COMMENTED** [review]
removing comments

**Review by scott-williams-az — COMMENTED** [review]
removing comments

**Review by maycomayco — COMMENTED** [review]
Looks good. I reviewed the changes, but I haven't approved them yet because I'm waiting for Michael to review them.

**Review by mlsamuelson — APPROVED** [review]

**Pull request merged**